--- CMakeLists.txt.orig	2026-01-04 13:47:42.682817670 +0300
+++ CMakeLists.txt	2026-01-04 13:48:26.411039996 +0300
@@ -29,2 +29,9 @@
+
+# C++ standard
+set(CMAKE_CXX_STANDARD 20)
+set(CMAKE_CXX_STANDARD_REQUIRED YES)
+
+# Compile commands
+set(CMAKE_EXPORT_COMPILE_COMMANDS YES)

 project(${APP_NAME})
@@ -87,3 +94,10 @@
 
 # Make sure the AXGameFinalSetup is included at the end of this file
 include(AXGameFinalSetup)
+
+add_custom_target(pipeline ALL
+    COMMAND python ${CMAKE_CURRENT_SOURCE_DIR}/Pipeline/pipeline_all.py
+    COMMENT "Running pipeline"
+)
+
+add_dependencies(${APP_NAME} pipeline)
